src/routes/_api/oauth.js

```diff
@@ -27,11 +27,11 @@ export function generateAuthLink (instanceName, clientId, redirectUri) {
 
 export function getAccessTokenFromAuthCode (instanceName, clientId, clientSecret, code, redirectUri) {
   const url = `${basename(instanceName)}/oauth/token`
-  return post(url, {
-    client_id: clientId,
-    client_secret: clientSecret,
-    redirect_uri: redirectUri,
-    grant_type: 'authorization_code',
-    code
-  }, null, { timeout: WRITE_TIMEOUT })
+  const data = new FormData();
+  data.append('client_id', clientId);
+  data.append('client_secret', clientSecret);
+  data.append('redirect_uri', redirectUri);
+  data.append('grant_type', 'authorization_code');
+  data.append('code', code);
+  return post(url, new URLSearchParams(data), null, { timeout: WRITE_TIMEOUT })
```

Comment:
Based on my reading of [the `URLSearchParams` constructor](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams), it does not actually care if you use `FormData` – any iterable of key-value string pairs will work here. So I find it a bit confusing to be using `FormData` here.